packages/shared/routes/setup/CustomNetwork.svelte

```diff
@@ -0,0 +1,55 @@
+<script lang="typescript">
+    import { localize } from '@core/i18n'
+    import { getOfficialNetworkConfig, INode, updateClientOptions } from '@core/network'
+    import { updateActiveProfileSettings } from '@core/profile'
+    import { appRouter } from '@core/router'
+    import { OnboardingLayout, Text, Button, Spinner } from 'shared/components'
+    import { NodeConfigurationForm } from 'shared/components/molecules'
+
+    let nodeConfigurationForm: NodeConfigurationForm
+    let nodeUrl: string
+    let isBusy = false
+
+    function onBackClick(): void {
+        $appRouter.previous()
+    }
+    function onSuccess(_isNetworkSwitch: boolean, node: INode, _oldNodeUrl: string): void {
```

Comment:
No longer allowing switching networks in the future so potentially we need to refactor this out.